recipe/meta.yaml

```diff
@@ -26,10 +28,10 @@ test:
     - if not exist %LIBRARY_PREFIX%\\bin\\ucrtbase.dll exit 1
 
 about:
-  home: https://developer.microsoft.com/en-us/windows/downloads/windows-10-sdk/
+  home: https://developer.microsoft.com/en-us/windows/downloads/windows-sdk/
   license: LicenseRef-Proprietary
   license_file: LICENSE.txt
-  summary: 'Redistributable files for Windows SDK. This is only needed Windows <10'
+  summary: Redistributable files for Windows SDK
```

Comment:
This has always confused me; https://github.com/conda-forge/vc-feedstock/pull/37 was using a win 2016 Server based image (`vs2017-win2016`); these have been dropped by azure >2 years ago already, though the SDK still has explicit support. However, I regularly see builds picking up ucrt-libraries (e.g. `api-ms-win-crt-*`) in the link check, so it doesn't seem like that's strictly win<10?

Or does the link check wrongly/redundantly pick the files in the environment, rather the ones provided by the OS? In that case, I guess this update would only be required for really old windows installations (>8years). Unless there are also some newer libraries in the SDK that are not provided on older win10 machines...

Reply:
Following that line of thought, comparing the output of the artefact inspection in this PR with the current [content](https://conda-metadata-app.streamlit.app/?q=conda-forge%2Fwin-64%2Fucrt-10.0.22621.0-h57928b3_0.tar.bz2) of our ucrt package shows that there's three extra DLLs in the newer SDK:
```
Library/bin/api-ms-win-core-fibers-l1-1-1.dll
Library/bin/api-ms-win-core-kernel32-legacy-l1-1-1.dll
Library/bin/api-ms-win-core-sysinfo-l1-2-0.dll
```
Presumably these are likewise not available in older windows installations, and we therefore need to keep shipping `ucrt` even after we (eventually) drop support for win<10. 🤔

Reply:
From https://learn.microsoft.com/en-us/uwp/win32-and-com/win32-apis

> This topic lists the Win32 APIs that are part of the Universal Windows Platform (UWP), and that are implemented by all Windows devices.
>
> For convenience, in the Microsoft Windows Software Development Kit (SDK) is an umbrella library named WindowsApp.lib. That umbrella library provides the exports for the set of Win32 APIs in question. To access those APIs in your app, link WindowsApp.lib (and no other libraries).

This sounds to me that the umbrella library is going to be the same but MS might choose to split some of the libraries out into smaller DLLs. Thus this probably still is only needed for win<10

Reply:
What I meant is something like
```
INFO (clang,Library/bin/clang.exe): Needed DSO Library/bin/api-ms-win-crt-runtime-l1-1-0.dll found in conda-forge/win-64::ucrt==10.0.22621.0=h57928b3_0
INFO (clang,Library/bin/clang.exe): Needed DSO Library/bin/api-ms-win-crt-heap-l1-1-0.dll found in conda-forge/win-64::ucrt==10.0.22621.0=h57928b3_0
INFO (clang,Library/bin/clang.exe): Needed DSO Library/bin/api-ms-win-crt-environment-l1-1-0.dll found in conda-forge/win-64::ucrt==10.0.22621.0=h57928b3_0
INFO (clang,Library/bin/clang.exe): Needed DSO Library/bin/api-ms-win-crt-math-l1-1-0.dll found in conda-forge/win-64::ucrt==10.0.22621.0=h57928b3_0
INFO (clang,Library/bin/clang.exe): Needed DSO Library/bin/api-ms-win-crt-string-l1-1-0.dll found in conda-forge/win-64::ucrt==10.0.22621.0=h57928b3_0
INFO (clang,Library/bin/clang.exe): Needed DSO Library/bin/api-ms-win-crt-stdio-l1-1-0.dll found in conda-forge/win-64::ucrt==10.0.22621.0=h57928b3_0
INFO (clang,Library/bin/clang.exe): Needed DSO Library/bin/api-ms-win-crt-utility-l1-1-0.dll found in conda-forge/win-64::ucrt==10.0.22621.0=h57928b3_0
INFO (clang,Library/bin/clang.exe): Needed DSO Library/bin/api-ms-win-crt-convert-l1-1-0.dll found in conda-forge/win-64::ucrt==10.0.22621.0=h57928b3_0
INFO (clang,Library/bin/clang.exe): Needed DSO Library/bin/api-ms-win-crt-time-l1-1-0.dll found in conda-forge/win-64::ucrt==10.0.22621.0=h57928b3_0
INFO (clang,Library/bin/clang.exe): Needed DSO Library/bin/api-ms-win-crt-filesystem-l1-1-0.dll found in conda-forge/win-64::ucrt==10.0.22621.0=h57928b3_0
INFO (clang,Library/bin/clang.exe): Needed DSO Library/bin/api-ms-win-crt-locale-l1-1-0.dll found in conda-forge/win-64::ucrt==10.0.22621.0=h57928b3_0
```

If those are false positives on Win>=10, then great!

Reply:
I think so, but the documentation is too sparse to fully confirm by myself.

Reply:
Just saw a confirmation of this kind of thing in the openssl feedstock. In particular, we see that the same library gets detected either from the sysroot or from our own environment (could be random, or that the link check works differently between DLLs & exes)

```
   INFO (openssl,Library/bin/openssl.exe):         Needed DSO                   Library/bin/api-ms-win-crt-stdio-l1-1-0.dll found in conda-forge/win-64::ucrt==10.0.22621.0=h57928b3_0
   INFO (openssl,Library/bin/libcrypto-3-x64.dll): Needed DSO C:/Windows/System32/downlevel/api-ms-win-crt-stdio-l1-1-0.dll found in $SYSROOT
   INFO (openssl,Library/bin/libssl-3-x64.dll):    Needed DSO C:/Windows/System32/downlevel/api-ms-win-crt-stdio-l1-1-0.dll found in $SYSROOT
```

Full output below

<details>

```
   INFO: sysroot: 'C:/Windows/' files: '['winhlp32.exe', 'win.ini', 'twain_32/wiatwain.ds', 'twain_32.dll']'
   INFO (openssl,Library/bin/openssl.exe): Library\bin\libssl-3-x64.dll found in this package
   INFO (openssl,Library/bin/openssl.exe): Library\bin\libcrypto-3-x64.dll found in this package
   INFO (openssl,Library/bin/openssl.exe): Needed DSO C:/Windows/System32/WS2_32.dll found in $SYSROOT
   INFO (openssl,Library/bin/openssl.exe): Needed DSO C:/Windows/System32/KERNEL32.dll found in $SYSROOT
   INFO (openssl,Library/bin/openssl.exe): Needed DSO Library/bin/VCRUNTIME140.dll found in conda-forge/win-64::vc14_runtime==14.40.33810=ha82c5b3_20
   INFO (openssl,Library/bin/openssl.exe): Needed DSO Library/bin/api-ms-win-crt-string-l1-1-0.dll found in conda-forge/win-64::ucrt==10.0.22621.0=h57928b3_0
   INFO (openssl,Library/bin/openssl.exe): Needed DSO Library/bin/api-ms-win-crt-convert-l1-1-0.dll found in conda-forge/win-64::ucrt==10.0.22621.0=h57928b3_0
   INFO (openssl,Library/bin/openssl.exe): Needed DSO Library/bin/api-ms-win-crt-stdio-l1-1-0.dll found in conda-forge/win-64::ucrt==10.0.22621.0=h57928b3_0
   INFO (openssl,Library/bin/openssl.exe): Needed DSO Library/bin/api-ms-win-crt-runtime-l1-1-0.dll found in conda-forge/win-64::ucrt==10.0.22621.0=h57928b3_0
   INFO (openssl,Library/bin/openssl.exe): Needed DSO Library/bin/api-ms-win-crt-filesystem-l1-1-0.dll found in conda-forge/win-64::ucrt==10.0.22621.0=h57928b3_0
   INFO (openssl,Library/bin/openssl.exe): Needed DSO Library/bin/api-ms-win-crt-utility-l1-1-0.dll found in conda-forge/win-64::ucrt==10.0.22621.0=h57928b3_0
   INFO (openssl,Library/bin/openssl.exe): Needed DSO Library/bin/api-ms-win-crt-environment-l1-1-0.dll found in conda-forge/win-64::ucrt==10.0.22621.0=h57928b3_0
   INFO (openssl,Library/bin/openssl.exe): Needed DSO Library/bin/api-ms-win-crt-time-l1-1-0.dll found in conda-forge/win-64::ucrt==10.0.22621.0=h57928b3_0
   INFO (openssl,Library/bin/openssl.exe): Needed DSO Library/bin/api-ms-win-crt-math-l1-1-0.dll found in conda-forge/win-64::ucrt==10.0.22621.0=h57928b3_0
   INFO (openssl,Library/bin/openssl.exe): Needed DSO Library/bin/api-ms-win-crt-heap-l1-1-0.dll found in conda-forge/win-64::ucrt==10.0.22621.0=h57928b3_0
   INFO (openssl,Library/bin/openssl.exe): Needed DSO Library/bin/api-ms-win-crt-locale-l1-1-0.dll found in conda-forge/win-64::ucrt==10.0.22621.0=h57928b3_0
   INFO (openssl,Library/bin/libcrypto-3-x64.dll): Needed DSO C:/Windows/System32/WS2_32.dll found in $SYSROOT
   INFO (openssl,Library/bin/libcrypto-3-x64.dll): Needed DSO C:/Windows/System32/ADVAPI32.dll found in $SYSROOT
   INFO (openssl,Library/bin/libcrypto-3-x64.dll): Needed DSO C:/Windows/System32/CRYPT32.dll found in $SYSROOT
   INFO (openssl,Library/bin/libcrypto-3-x64.dll): Needed DSO C:/Windows/System32/USER32.dll found in $SYSROOT
   INFO (openssl,Library/bin/libcrypto-3-x64.dll): Needed DSO C:/Windows/System32/KERNEL32.dll found in $SYSROOT
   INFO (openssl,Library/bin/libcrypto-3-x64.dll): Needed DSO C:/Windows/System32/VCRUNTIME140.dll found in $SYSROOT
   INFO (openssl,Library/bin/libcrypto-3-x64.dll): Needed DSO C:/Windows/System32/downlevel/api-ms-win-crt-stdio-l1-1-0.dll found in $SYSROOT
   INFO (openssl,Library/bin/libcrypto-3-x64.dll): Needed DSO C:/Windows/System32/downlevel/api-ms-win-crt-convert-l1-1-0.dll found in $SYSROOT
   INFO (openssl,Library/bin/libcrypto-3-x64.dll): Needed DSO C:/Windows/System32/downlevel/api-ms-win-crt-string-l1-1-0.dll found in $SYSROOT
   INFO (openssl,Library/bin/libcrypto-3-x64.dll): Needed DSO C:/Windows/System32/downlevel/api-ms-win-crt-time-l1-1-0.dll found in $SYSROOT
   INFO (openssl,Library/bin/libcrypto-3-x64.dll): Needed DSO C:/Windows/System32/downlevel/api-ms-win-crt-utility-l1-1-0.dll found in $SYSROOT
   INFO (openssl,Library/bin/libcrypto-3-x64.dll): Needed DSO C:/Windows/System32/downlevel/api-ms-win-crt-heap-l1-1-0.dll found in $SYSROOT
   INFO (openssl,Library/bin/libcrypto-3-x64.dll): Needed DSO C:/Windows/System32/downlevel/api-ms-win-crt-runtime-l1-1-0.dll found in $SYSROOT
   INFO (openssl,Library/bin/libcrypto-3-x64.dll): Needed DSO C:/Windows/System32/downlevel/api-ms-win-crt-filesystem-l1-1-0.dll found in $SYSROOT
   INFO (openssl,Library/bin/libcrypto-3-x64.dll): Needed DSO C:/Windows/System32/downlevel/api-ms-win-crt-environment-l1-1-0.dll found in $SYSROOT
   INFO (openssl,Library/bin/libssl-3-x64.dll): Library\bin\libcrypto-3-x64.dll found in this package
   INFO (openssl,Library/bin/libssl-3-x64.dll): Needed DSO C:/Windows/System32/WS2_32.dll found in $SYSROOT
   INFO (openssl,Library/bin/libssl-3-x64.dll): Needed DSO C:/Windows/System32/KERNEL32.dll found in $SYSROOT
   INFO (openssl,Library/bin/libssl-3-x64.dll): Needed DSO C:/Windows/System32/VCRUNTIME140.dll found in $SYSROOT
   INFO (openssl,Library/bin/libssl-3-x64.dll): Needed DSO C:/Windows/System32/downlevel/api-ms-win-crt-heap-l1-1-0.dll found in $SYSROOT
   INFO (openssl,Library/bin/libssl-3-x64.dll): Needed DSO C:/Windows/System32/downlevel/api-ms-win-crt-environment-l1-1-0.dll found in $SYSROOT
   INFO (openssl,Library/bin/libssl-3-x64.dll): Needed DSO C:/Windows/System32/downlevel/api-ms-win-crt-runtime-l1-1-0.dll found in $SYSROOT
   INFO (openssl,Library/bin/libssl-3-x64.dll): Needed DSO C:/Windows/System32/downlevel/api-ms-win-crt-time-l1-1-0.dll found in $SYSROOT
   INFO (openssl,Library/bin/libssl-3-x64.dll): Needed DSO C:/Windows/System32/downlevel/api-ms-win-crt-string-l1-1-0.dll found in $SYSROOT
   INFO (openssl,Library/bin/libssl-3-x64.dll): Needed DSO C:/Windows/System32/downlevel/api-ms-win-crt-utility-l1-1-0.dll found in $SYSROOT
   INFO (openssl,Library/bin/libssl-3-x64.dll): Needed DSO C:/Windows/System32/downlevel/api-ms-win-crt-filesystem-l1-1-0.dll found in $SYSROOT
   INFO (openssl,Library/bin/libssl-3-x64.dll): Needed DSO C:/Windows/System32/downlevel/api-ms-win-crt-convert-l1-1-0.dll found in $SYSROOT
   INFO (openssl,Library/bin/libssl-3-x64.dll): Needed DSO C:/Windows/System32/downlevel/api-ms-win-crt-stdio-l1-1-0.dll found in $SYSROOT
```
</details>